.nvmrc

```
@@ -1 +1 @@
-lts/dubnium
+lts/fermium
```

Fermium is Node 14. Node 12 is erbium. https://nodejs.org/en/about/releases/

```suggestion
lts/erbium
```

---

Just to be clear, this issue is to update to `erbium`? Because the issue description says `lts/fermium`;

---

If this is for upgrading to Node 12, it should be `erbium`. I must have carelessly copied/pasted the description from #78  to #77 without updating the Node version in the description. The issue description is now updated.